jetty-http/src/test/java/org/eclipse/jetty/http/HttpParserTest.java

```diff
@@ -422,7 +427,7 @@ public void testTrailingSpacesInHeaderNameInCustom0Mode() throws Exception
         Assert.assertEquals("Other", _hdr[1]);
         Assert.assertEquals("value", _val[1]);
 
-        Assert.assertThat(_complianceViolation, contains(HttpComplianceSection.NO_WS_AFTER_FIELD_NAME));
+        Assert.assertThat(_complianceViolation, contains(HttpComplianceSection.NO_WS_AFTER_FIELD_NAME,HttpComplianceSection.NO_WS_AFTER_FIELD_NAME));
```

Comment:
you could use static imports to increase the code readability
`import static org.eclipse.jetty.http.HttpComplianceSection. NO_WS_AFTER_FIELD_NAME`